.gitlab-ci.yml

```
@@ -18,24 +28,24 @@ before_script:
 validate_doc:
   stage: test
   script:
-    - npm exec -- bump deploy --dry-run bump.openapi.v3.yml
+    - npm exec -- bump deploy --dry-run 'bump.openapi.v3.yml' --doc documentation_slug --token bump_token
   only:
     - branches
   except:
-    - master
+    - main
 
 deploy_doc:
   stage: deploy
   script:
-    - npm exec -- bump deploy bump.openapi.v3.yml
+    - npm exec -- bump deploy 'bump.openapi.v3.yml' --doc documentation_slug --token bump_token
   only:
-    - master
+    - main
 
 diff_doc:
   stage: test
   script:
-    - ./.gitlab/diff-comment-mr.sh bump.openapi.v3.yml
+    - ./.gitlab/diff-comment-mr.sh 'bump.openapi.v3.yml' bump_token
```

Instead of adding a comment in the sh script, you can call sh here directly which removes the necessity for the execution flag on the script file:

```suggestion
    - sh ./.gitlab/diff-comment-mr.sh "bump.openapi.v3.yml" "bump_documentation_slug" "bump_token"
```

---

Thanks for suggestion, I'd favor both:
- update comment (not added  by this PR) to be correct  with 3 parameters
- as suggested, edit where script is called